src/components/Stack/README.md

```diff
@@ -138,6 +138,19 @@ Use for aligning buttons or secondary content to the right edge of another eleme
 
 ---
 
+## Stack item
+
+The Stack will implicitly wrap each of its children in a Stack.Item unless used directly. Use Stack.Item as a subcomponent with the fill prop to have the item fill the available horizontal space in the stack. See "Stack where a single item fills the remaining space" as an example of this above.
```

Comment:
The Stack will implicitly wrap each of its children in a Stack.Item unless used directly 👍 I'm glad we've finally gotten this documented ❤️

Reply:
Yes! I'm also happy to see this get documented!

I don't think we use determiners before component names, unless it's followed by the word "component". So, something like `The stack component will implicitly...`.

In documentation, we also use sentence case for components, ie: a stack item vs. Stack.Item. https://docs.google.com/document/d/1zVDsHIWhoir2svRjdtSdRbD_ruTz3K1nAJcQLGPVQwM/edit#heading=h.lm3ja0wzryt7

It might be a good idea to link to the example using a named anchor. The resource list documentation has an example of this.